src/Bowling.js

```diff
@@ -0,0 +1,104 @@
+var Scorecard = function() {
+  this.frames = [[],[],[],[],[],[],[],[],[],[]];
```

Comment:
This is fine for now as it works, but feels like there could be a more elegant solution - a separate Frame object? Something that holds a reference to a frame, or that collects scores as you go along?